habit/src/styles/common.js

```diff
@@ -8,10 +8,7 @@ module.exports = StyleSheet.create({
         justifyContent: 'center',
         alignItems: 'center',
     },
-    // TODO KARAN Move this to styles/login.js
-    loginContainer: {
-        width: '100%',
```

Comment:
Do we not need this now?

Reply:
no i have moved the final style to the relevant file

Reply:
I was talking about width